autoload/lsp.vim

```diff
@@ -135,7 +137,7 @@ function! s:register_events() abort
         autocmd BufReadPost * call s:on_text_document_did_open()
         autocmd BufWritePost * call s:on_text_document_did_save()
         autocmd BufWinLeave * call s:on_text_document_did_close()
-        autocmd InsertLeave * call s:on_text_document_did_change()
+        autocmd InsertLeave,TextChanged * call s:on_text_document_did_change()
```

Comment:
would need to debounce `TextChanged` as it could slow down vim due to did change. When we implement incremental change it would be easier. But there is a PR currently for it at https://github.com/prabirshrestha/vim-lsp/pull/188